PAPI_FAQ.html

```diff
@@ -68,16 +76,16 @@ <h2>The PAPI Library</h2>
 <h2>The PAPI GIT Source Repository</h2>
 
 <strong>Can I browse the source repository on the web?</strong>
-<p>Yes. The latest copy of the PAPI source tree is viewable through a web based source browser accessible from the GitHub repository at:<br /><a href="../../trac/papi/browser">https://github.com/icl-utk-edu/papi</a></p>
+<p>Yes. The latest copy of the PAPI source tree is viewable through a web based source browser accessible from the GitHub repository at:<br /><a href="https://github.com/icl-utk-edu/papi">https://github.com/icl-utk-edu/papi</a></p>
 <p>&nbsp;</p><strong>How do I download a copy of the current PAPI source tree?</strong>
-<p>Make sure git is installed on your machine. You can download a copy <a href="http://git-scm.com/">here</a>.</p>
+<p>Make sure git is installed on your machine. You can download a copy of git <a href="http://git-scm.com/">here</a>.</p>
```

Comment:
I would recommend changing this to a direct link to the actual git download section: https://git-scm.com/downloads. Currently, it just goes to the main git landing page.